static/js/gtm.js

```diff
@@ -0,0 +1,47 @@
+window.dataLayer = window.dataLayer || [];
+window.dataLayer.push({
```

Comment:
switch the order?

```
function gtag() { dataLayer.push(arguments); }
gtag({
  'gtm.start': new Date().getTime(),
  event: 'gtm.js'
});
```

Reply:
I don't think that is the same.. `arguments` behaves as an array..

Reply:
/facepalm